app/scripts/xprizeDemo.xml

```diff
@@ -0,0 +1,217 @@
+<!-- Copyright (C) 2021 Fondazione Istituto Italiano di Tecnologia (IIT) -->
+<!-- All Rights Reserved. -->
+<!-- Author: Giulio Romualdi <giulio.romualdi@iit.it> -->
+
+<application>
+
+  <name>Xrprize Demo (Virtualizer Xsens SenseGlove)</name>
+  <description>Application to run the ANA Avatar XPRIZE demo.</description>
+  <version>1.0</version>
+  <authors>
+    <author email="giulio.romualdi@iit.it">Giulio Romualdi</author>
+  </authors>
+
+  <!-- Modules -->
+  <!-- icub-head -->
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --from camera/ServerGrabberDualDragon.ini --split true</parameters>
+    <node>icub-head</node>
+  </module>
+
+  <module>
+    <name>WalkingModule</name>
+    <node>icub-head</node>
+    <parameters>--from dcm_walking_joint_retargeting.ini</parameters>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --device AudioRecorderWrapper --subdevice portaudioRecorder --name /icub/microphone  --min_samples_over_network 4000  --max-samples_over_network 4000  --rate 160000 --samples 16000</parameters>
+    <node>icub-head</node>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters>--device AudioPlayerWrapper --subdevice portaudioPlayer --name /icub/speakers --playback_network_buffer_lenght 0.4</parameters>
+    <node>icub-head</node>
+    <dependencies>
+      <port timeout="5.0" request="start" reply="[ok]">/icub-virtualizer/microphone/rpc</port>
+    </dependencies>
+  </module>
+
+  <!-- icub-console -->
+
+  <module>
+    <name>camCalibWithPose</name>
+    <parameters>--context cameraCalibration --from icubEyes.ini --group CAMERA_CALIBRATION_LEFT --name /icub/camcalib/left --useIMU</parameters>
```

Comment:
Need to check how the `--useIMU` option affects running the module on the robots equipped with IMUs streaming through MAS interface, for example `iCubGenova09`.

Reply:
Apparently using `useIMU` option with `iCubGenova09` does not work. On removing that option, the module starts streaming the images.